File: src/main/java/net/bms/genera/init/GeneraTileEntities.java
package net.bms.genera.init;

import net.bms.genera.te.TileFaerieHome;
import net.minecraftforge.fml.common.registry.GameRegistry;

/**
 * Created by ben on 4/3/17.
 */
public class
GeneraTileEntities {

    public static void init() {
        GameRegistry.registerTileEntity(TileFaerieHome.class, "faerie_home");
    }

}

File: src/main/kotlin/net/bms/genera/init/GeneraTileEntities.kt
package net.bms.genera.init

import net.bms.genera.te.TileFaerieHome
import net.minecraftforge.fml.common.registry.GameRegistry

/**
 * Created by ben on 4/3/17.
 */
object GeneraTileEntities {
    fun initTileEntities() {
        GameRegistry.registerTileEntity(TileFaerieHome::class.java, "faerie_enclosure")
    }
}
